apps/web/src/pages/EmployeeProfile/messages.ts

```diff
@@ -1,11 +1,9 @@
 import { defineMessages } from "react-intl";
 
+import messages from "~/messages/profileMessages";
+
 export default defineMessages({
-  careerPlanning: {
-    defaultMessage: "Career planning",
-    id: "zN7MBv",
-    description: "Title for a users career plan",
-  },
+  careerPlanning: messages.careerPlanning,
```

Comment:
I think this pattern could cause problems. I would recommend moving the string to `packages/i18n/src/messages/commonMessages.ts` as `commonMessages.careerPlanning` since this is used in multiple contexts.

Reply:
It actually did cause problems so I removed it from there https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/3fca50e63234d9876225b3a8f88ff69eaa4facc9

Reply:
Yep, still should move it from `profileMessages` to `commonMessages` though as it will be easier to find and will likely be used again.

Reply:
Sure, why not https://github.com/GCTC-NTGC/gc-digital-talent/pull/12607/commits/572a32ba5435dce9ee85c005a6dc7313bfecef96
Where we've been placing reusable messages I feel we've played it loosely